sumpy/symbolic.py

```diff
@@ -241,4 +241,53 @@ def map_subscript(self, expr):
         else:
             self.raise_conversion_error(expr)
 
+    def map_call(self, expr):
+        if expr.function.name == "hankel_1":
+            args = [self.rec(param) for param in expr.parameters]
+            args.append(0)
+            return Hankel1(*args)
+        elif expr.function.name == "bessel_j":
+            args = [self.rec(param) for param in expr.parameters]
+            args.append(0)
+            return BesselJ(*args)
+        else:
+            return PymbolicToSympyMapper.map_call(self, expr)
+
+
+import sympy
+
+
+class _BesselOrHankel(sympy.Function):
+    """A symbolic function for BesselJ or Hankel1 functions
+    that keeps track of the derivatives taken of the function
+    arguments are (order, z, nderivs)"""
+    nargs = (3,)
+
+    def fdiff(self, argindex=1):
+        if argindex in (1, 3):
+            # we are not differentiating w.r.t order or nderivs
+            return 0
```

Comment:
```suggestion
            raise ValueError()
```
?
(Is this code path ever hit? I feel much better about errors than silent garbage data.)

Reply:
Not sure. Let's try